04LessonFour/in-class-project/css/style.css

```diff
@@ -1,84 +1,22 @@
 body {
-    background-color: skyblue;
-}
-
-.head {
-    width: 100%;
-    height: 200px;
-    border: solid 3px black;
-}
-
-.head img {
-    height: 100px;
-    width: 100px;
-    margin-left: 655px;
-    border: solid 2px black;
-}
-
-.head h1 {
-    text-align: center;
-    margin-top: 10px;
-}
-
-nav a {
-    font-family: sans-serif;
-    font-size: 15px;
+    margin: auto;
+    width: 80%;
+    font-family: Helvetica, Arial, sans-serif;
+    margin-bottom: 10px;
+}
+#header {
+    font-size: 24px;
+    background-color: #0071c5;
     color: white;
-    text-align: center;
-    margin-left: 300px;
-}
-
-#two {
-    border: solid 2px black;
-    width: 60%;
-    float: left;
-}
-
-#two h2 {
-    text-align: center;
 }
-
-#two h3 {
-    text-decoration: underline;
+.support {
+    background-color: grey;
+    margin-top: -6x;
+    Padding-left: 10px;

```

Comment:
"Padding-left" is not being recognized. Simply change the "P" to a "p" - "padding-left" should work here.